pkg/features/kube_features.go

```diff
@@ -470,7 +470,7 @@ const (
 
 	// owner: @zshihang
 	// kep: https://kep.k8s.io/2800
-	// beta: v1.24
+	// ga: v1.26
```

Comment:
keep the beta line and add the GA line after it

Reply:
done